examples/reporting/campaign_report_to_csv.rb

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/ruby
+# Encoding: utf-8
+#
+# Copyright:: Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# This code example illustrates how to get metrics about a campaign and 
+# serialize the result as a CSV file.
+
+require 'optparse'
+require 'time'
+require 'csv'
+require 'google/ads/google_ads'
+
+PAGE_SIZE = 50
```

Comment:
Please make this pagesize considerably bigger, since we are dealing with reports which usually will have lots of rows.